### PR TITLE
reverse-string: add to track

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Add `reverse-string` to track.

### CODE REVIEW

**Comment by sjwarner-bp** [comment]
Hi all, I am not too sure why the build is failing - it says

`FAILURES:  reverse-string (FileNotFound)`

Does this mean that I have not added reverse-string to a particular list of exercises somewhere?

File: config.json [hunk 1]
```diff
@@ -27,6 +27,16 @@
         "logic"
       ]
     },
+    {
+      "uuid": "",
```
Comment (cmccandless) [inline config.json]: You need to add a unique UUID. Use `configlet uuid` to generate one.

Comment (cmccandless) [reply config.json]: Also, make sure when reading the Travis build output that you look for the earliest error; sometimes these cause the later ones and are the real problem. From this particular failure:
```Bash
$ ./bin/configlet lint .

-> The exercise 'reverse-string' was found in config.json, but does not have a UUID.

The command "./bin/configlet lint ." exited with 1.
```

**Comment by sjwarner-bp** [comment]
I've added the uuid - silly oversight on my part! The build is still failing with the same error, and I can't seem to see any other errors on the output?

**Comment by cmccandless** [comment]
Current changes look good. Please remove "[WIP]" from the PR title if you are finished working on this.

**Comment by sjwarner-bp** [comment]
Bump! 🙂

**Comment by cmccandless** [comment]
@sjwarner-bp Merged! Thanks for working on this!

**Pull request merged**